pages/index.js

```diff
@@ -176,7 +180,7 @@ class Index extends React.Component {
 
                     <div className='usage'>
                         <h2> Usage </h2>
-                        <p>The following is an example of a matrix multiplication on encrypted data using TF Encrypted</p>
+                        <p>TF Encrypted looks just like the tools you already know.  Here is a simple classification model:</p>
```

Comment:
suggest changing to "simple neural network" (since the task is kind of undefined here)